agent/src/test/java/com/appland/appmap/record/RecorderTest.java

```diff
@@ -98,6 +104,78 @@ public void testUnwriteableOutputFile() throws IOException {
     assertTrue(exception.toString().indexOf("java.lang.RuntimeException: ") == 0);
   }
 
+  @Test
+  // for Files.move when: REPLACE_EXISTING, ATOMIC_MOVE
+  public void testWriteFileAcrossFilesystems() throws IOException {
+    if (!System.getProperty("os.name").equals("Linux")) {
+        //  /dev/shm exists only on Linux
+        return;
```

Comment:
```suggestion
import static org.hamcrest.CoreMatchers.is;
import static org.junit.Assume.assumeThat;
...
  public void testWriteFileAcrossFilesystems() throws IOException {
    assumeThat(System.getProperty("os.name"), is("Linux"));
```

It would be idiomatic to use a [JUnit assumption](https://github.com/junit-team/junit4/wiki/Assumptions-with-assume) here.

Reply:
Changed to use `assumeTrue`.  `assumeThat` required `is` and the imports I tried for `is` didn't seem to include it.

Reply:
That's weird. Did you use the dependencies I mentioned in my suggestion? They worked for me.

Reply:
I used
```
import static org.junit.Assume.*
```
from the example in the [link](https://github.com/junit-team/junit4/wiki/Assumptions-with-assume) above.  I get
```
> Task :agent:compileTestJava FAILED
/home/test/src/appmap-java/agent/src/test/java/com/appland/appmap/record/RecorderTest.java:120: error: cannot find symbol
    assumeThat(System.getProperty("os.name"), is("Linux"));
                                              ^
  symbol:   method is(String)
  location: class RecorderTest
1 error
```
If it makes a difference (I believe it shouldn't) this is with Java 11.

Reply:
You need this one, too: 

```java
import static org.hamcrest.CoreMatchers.is;
```